lib/cocoapods/generator/xcconfig/xcconfig_helper.rb

```diff
@@ -165,13 +173,13 @@ def self.add_spec_build_settings_to_xcconfig(consumer, xcconfig)
         #
         # @return [void]
         #
-        def self.add_framework_build_settings(framework_path, xcconfig, sandbox_root)
+        def self.add_framework_build_settings(framework_path, xcconfig, sandbox_root, include_other_ldflags)
```

Comment:
this either needs to be a default argument or the other callsites need to be updated -- see the test failures